core-web/apps/dotcms-ui/src/app/portlets/dot-categories/dot-categories-create-edit/dot-categories-create-edit.component.html

```diff
@@ -0,0 +1,23 @@
+<dot-portlet-base *ngIf="vm$ | async as vm" [boxed]="false">
+    <div class="categories-create__tab-container">
+        <p-tabView class="categories-create__tabs">
+            <p-tabPanel [header]="'message.categories.tab.childrens' | dm">
+                <ng-template pTemplate="content">
+                    <dot-categories-list
+                        (updateCategory)="updateCategory($event)"
+                    ></dot-categories-list>
+                </ng-template>
+            </p-tabPanel>
+            <p-tabPanel [header]="'message.categories.tab.properties' | dm">
+                Properties works
```

Comment:
??

Reply:
It is Properties Tab in Categories Component.
![image](https://user-images.githubusercontent.com/113915849/193068602-ec059760-6006-4b11-9ee2-9f720d3609ab.png)